src/azul/deployment.py

```diff
@@ -285,13 +284,11 @@ class Terraform:
 
     @cached_property
     def taggable_resources(self) -> Sequence[str]:
-        schema = subprocess.run(['terraform', 'providers', 'schema', '-json'],
-                                cwd=os.path.join(os.environ['project_root'], 'terraform'),
-                                check=True,
-                                capture_output=True,
-                                text=True,
-                                shell=False)
-        schema = json.loads(schema.stdout)
+        schema_path = os.path.join(os.environ['project_root'],
+                                   'terraform',
+                                   '_schema.json')
```

Comment:
hard-coded file name

Reply:
Is there a preferred way to extract this? Potential approaches I can think of:

* Extract the path into a variable in `environment`
* Define the path as a constant in `src.azul.deployment` and reference that path in `scripts/terraform_schema.py`
* Require path to be provided as an argument in `scripts/terraform_schema.py` (I chose this initially, but I can't think of a simple way to provide the same path to `src.azul.deployment` with this approach)

Reply:
Hannes prefers approach 1.